topydo/lib/Config.py

```diff
@@ -186,6 +193,49 @@ def hidden_tags(self):
         hidden_tags = self.cp.get('ls', 'hide_tags')
         return [] if hidden_tags == '' else hidden_tags.split(',')
 
+    def priority_colors(self):
+        """ Returns a dict with priorities as keys and color numbers as value. """
+        pri_colors_str = self.cp.get('colorscheme', 'priority_colors')
+
+        def _str_to_dict(p_string):
+            pri_colors_dict = dict()
+            for pri_color in pri_colors_str.split(','):
```

Comment:
> Corresponding test-case: use an empty value for `priority_colors`

Done. However I got some afterthoughts and I don't know whether current behavior for `priority_colors` is intuitive enough. As You can see in revised tests, empty value for `priority_colors` in config will give us default results, but empty values for other `*_color` parameters will return `NEUTRAL_COLOR`. I'm thinking about unifying this, so empty `priority_colors ` would also return `NEUTRAL_COLOR` for A, B, and C. What do You think?